sdk/file_sdk/CompoundFileFormat/CompoundFileFormat.pro

```diff
@@ -5,6 +5,7 @@ TARGET    = modcompoundfile
 TEMPLATE = lib
 CONFIG += staticlib warn_on c++11 debug_and_release
 INCLUDEPATH += $$REPO_ROOT/sdk/rmscrypto_sdk/CryptoAPI $$REPO_ROOT/sdk/file_sdk
+INCLUDEPATH += $$REPO_ROOT/sdk/rms_sdk/Profile
```

Comment:
nit: see if it fits in the line above with the other paths, if not leave it as is

Reply:
Doesn't fit.